Validation/HGCalTriggerPrimitives/plugins/HGCalTrigValidator.cc

```diff
@@ -0,0 +1,309 @@
+  h_cl_mipPt = iBooker.book1D("cl_mipPt","cluster2D mipPt",600, 0, 600);
+  h_cl_pt = iBooker.book1D("cl_pt","cluster2D pt",20, 0, 20);
+  h_cl_layer = iBooker.book1D("cl_layer","cluster2D layer", 50, 0, 50);
+  h_cl_subdet = iBooker.book1D("cl_subdet","cluster2D subdet", 4, 2, 6); 
+  h_cl_cells_n = iBooker.book1D("cl_cells_n","cluster2D cells_n", 16, 0, 16);
+
+  // multiclusters
+  h_cl3d_energy = iBooker.book1D("cl3d_energy","cl3duster3D energy; energy", 80, 0, 80);
+  h_cl3d_eta = iBooker.book1D("cl3d_eta","cl3duster3D eta; eta", 8, -4, 4);
+  h_cl3d_phi = iBooker.book1D("cl3d_phi","cl3duster3D phi; phi", 8, -4, 4);
+  h_cl3d_n = iBooker.book1D("cl3d_n","cl3duster3D number", 12, 0, 12);
+  h_cl3d_pt = iBooker.book1D("cl3d_pt","cl3duster3D pt", 50, 0, 50);
+  h_cl3d_clusters_n = iBooker.book1D("cl3d_clusters_n","cl3duster3D clusters_n", 30, 0, 30);
+  // cluster shower shapes
+  h_cl3d_showerlength = iBooker.book1D("cl3d_showerlength","cl3duster3D showerlength", 50, 0, 50);
+  h_cl3d_coreshowerlength = iBooker.book1D("cl3d_coreshowerlength","cl3duster3D coreshowerlength", 16, 0, 16);
+  h_cl3d_firstlayer = iBooker.book1D("cl3d_firstlayer","cl3duster3D firstlayer", 50, 0, 50);
+  h_cl3d_maxlayer = iBooker.book1D("cl3d_maxlayer","cl3duster3D maxlayer", 50, 0, 50);
+  h_cl3d_seetot = iBooker.book1D("cl3d_seetot","cl3duster3D seetot", 50, 0, 0.05);
+  h_cl3d_seemax = iBooker.book1D("cl3d_seemax","cl3duster3D seemax", 40, 0, 0.04);
+  h_cl3d_spptot = iBooker.book1D("cl3d_spptot","cl3duster3D spptot", 800, 0, 0.08);
+  h_cl3d_sppmax = iBooker.book1D("cl3d_sppmax","cl3duster3D sppmax", 800, 0, 0.08);
+  h_cl3d_szz = iBooker.book1D("cl3d_szz","cl3duster3D szz", 50, 0, 50);
+  h_cl3d_srrtot = iBooker.book1D("cl3d_srrtot","cl3duster3D srrtot", 800, 0, 0.008);
+  h_cl3d_srrmax = iBooker.book1D("cl3d_srrmax","cl3duster3D srrmax", 900, 0, 0.009);
+  h_cl3d_srrmean = iBooker.book1D("cl3d_srrmean","cl3duster3D srrmean", 800, 0, 0.008);
+  h_cl3d_emaxe = iBooker.book1D("cl3d_emaxe","cl3duster3D emaxe", 15, 0, 1.5);
+  h_cl3d_bdteg = iBooker.book1D("cl3d_bdteg","cl3duster3D bdteg", 30, -0.7, 0.4);
+  h_cl3d_quality = iBooker.book1D("cl3d_quality","cl3duster3D quality", 20, 0, 2);
+
+  // towers
+  h_tower_n = iBooker.book1D("tower_n","tower n", 400, 1200, 1600);
+  h_tower_pt = iBooker.book1D("tower_pt","tower pt", 50, 0, 50);
+  h_tower_energy = iBooker.book1D("tower_energy","tower energy", 200, 0, 200);
+  h_tower_eta = iBooker.book1D("tower_eta","tower eta", 8, -4, 4); 
+  h_tower_phi = iBooker.book1D("tower_phi","tower phi", 8, -4, 4);
+  h_tower_etEm = iBooker.book1D("tower_etEm","tower etEm",  50, 0, 50);
+  h_tower_etHad = iBooker.book1D("tower_etHad","tower etHad", 30, 0, 0.3);
+  h_tower_iEta = iBooker.book1D("tower_iEta","tower iEta", 20, 0, 20);
+  h_tower_iPhi = iBooker.book1D("tower_iPhi","tower iPhi", 80, 0, 80);
+}
+
+void HGCalTrigValidator::analyze(const edm::Event& iEvent, const edm::EventSetup& iSetup){
+  using namespace edm;
```

Comment:
Later in the function `edm::` prefix is used. So in that case it is not needed to  add `using namespace edm`